src/module/migration/migrations/866-link-to-actor-size-again.ts

```diff
@@ -0,0 +1,30 @@
+import type { ActorSourcePF2e } from "@actor/data/index.ts";
+import { MigrationBase } from "../base.ts";
+import { isActorTypeSizeLinkable } from "@scene/helpers.ts";
+
+export class Migration866LinkToActorSizeAgain extends MigrationBase {
+    static override version = 0.866;
+
+    override async updateActor(actorSource: ActorSourcePF2e): Promise<void> {
+        if (isActorTypeSizeLinkable(actorSource.type)) return;
+
+        if (actorSource.prototypeToken.flags.pf2e) {
+            actorSource.prototypeToken.flags.pf2e.linkToActorSize = false;
+            actorSource.prototypeToken.flags.pf2e.autoscale = false;
+        }
+    }
+
+    override async updateToken(tokenSource: foundry.documents.TokenSource): Promise<void> {
```

Comment:
`updateToken` will include the actor if you need it.

Reply:
True.